openapi-bal-service/src/main/java/io/ballerina/openapi/converter/utils/ConverterCommonUtils.java

```diff
@@ -489,6 +489,6 @@ public static boolean containErrors(List<Diagnostic> diagnostics) {
     }
 
     public static String unescapeIdentifier(String parameterName) {
-        return parameterName.replaceAll("\\\\", "");
+        return parameterName.replaceAll("\\\\", "").replaceAll("'", "");
```

Comment:
I'm afraid to use runtime APIs since this tool work in compile-time, unable to access runtime details when the code generation is processed.  
Yeah, need to check the Unicode character as well :). I will add the test for it.

Reply:
+1